cluster/manifests/spot-node-rescheduler/rbac.yaml

```diff
@@ -0,0 +1,40 @@
+{{ if eq .ConfigItems.spot_node_rescheduler "true" }}
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: spot-node-rescheduler
+  namespace: kube-system
+  annotations:
+    iam.amazonaws.com/role: "{{ .Cluster.LocalID }}-spot-node-rescheduler"
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: spot-node-rescheduler
+rules:
+- apiGroups:
+  - ""
+  resources:
+  - pods
+  - nodes
+  verbs:
+  - get
+  - watch
+  - list
+  - update
+  - patch
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: spot-node-rescheduler-privileged-psp
```

Comment:
I don't think we need privileged psp. This should be a binding to the `spot-node-rescheduler` `ClusterRole` as defined above.